### PR TITLE
[SAIServer]Upgrade SAI server init script

### PR DESCRIPTION
**Pull request by richardyu-ms**
#### Why I did it
why
In order to apply different config across different platform, and use the code with a unified format, reuse syncd init script to init saiserver.

#### How I did it
how
Reuse syncd init script

#### How to verify it

Test
Test in DUT s6000 and dx010 with sonic 202205
```
ptf --test-dir ptf saisanity.L2SanityTest --relax --xunit --xunit-dir "/tmp/sai_qualify/test_results_tmp" --interface '0-0@eth0' --interface '0-1@eth1' --interface '0-2@eth2' --interface '0-3@eth3' --interface '0-4@eth4' --interface '0-5@eth5' --interface '0-6@eth6' --interface '0-7@eth7' --interface '0-8@eth8' --interface '0-9@eth9' --interface '0-10@eth10' --interface '0-11@eth11' --interface '0-12@eth12' --interface '0-13@eth13' --interface '0-14@eth14' --interface '0-15@eth15' --interface '0-16@eth16' --interface '0-17@eth17' --interface '0-18@eth18' --interface '0-19@eth19' --interface '0-20@eth20' --interface '0-21@eth21' --interface '0-22@eth22' --interface '0-23@eth23' --interface '0-24@eth24' --interface '0-25@eth25' --interface '0-26@eth26' --interface '0-27@eth27' --interface '0-28@eth28' --interface '0-29@eth29' --interface '0-30@eth30' --interface '0-31@eth31' --relax "--test-params=thrift_server='$DUTIP';port_config_ini='/tmp/sai_qualify/sai_test/resources/port_config.ini'"

ptf --test-dir test/sai_test sai_sanity_test.SaiSanityTest --relax --xunit --xunit-dir "/tmp/sai_qualify/test_results_tmp" --interface '0-0@eth0' --interface '0-1@eth1' --interface '0-2@eth2' --interface '0-3@eth3' --interface '0-4@eth4' --interface '0-5@eth5' --interface '0-6@eth6' --interface '0-7@eth7' --interface '0-8@eth8' --interface '0-9@eth9' --interface '0-10@eth10' --interface '0-11@eth11' --interface '0-12@eth12' --interface '0-13@eth13' --interface '0-14@eth14' --interface '0-15@eth15' --interface '0-16@eth16' --interface '0-17@eth17' --interface '0-18@eth18' --interface '0-19@eth19' --interface '0-20@eth20' --interface '0-21@eth21' --interface '0-22@eth22' --interface '0-23@eth23' --interface '0-24@eth24' --interface '0-25@eth25' --interface '0-26@eth26' --interface '0-27@eth27' --interface '0-28@eth28' --interface '0-29@eth29' --interface '0-30@eth30' --interface '0-31@eth31' --relax "--test-params=thrift_server='$DUTIP';port_config_ini='/tmp/sai_qualify/sai_test/resources/port_config.ini'"
```
#### Which release branch to backport (provide reason below if selected)

Todo:
Copy and change code automatically from syncd submodules

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Review by Gfrom2016 — APPROVED** [review]

File: platform/broadcom/docker-saiserver-brcm/syncd_init_common.sh [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    if [ x"$WARM_BOOT" == x"true" ]; then
+        CMD_ARGS+=" -t warm"
+    elif [ x"$FAST_REBOOT" == x"yes" ]; then
+        CMD_ARGS+=" -t fast"
+    elif [ x"$FASTFAST_REBOOT" == x"yes" ]; then
+        CMD_ARGS+=" -t fastfast"
+    fi
+}
+
+config_syncd_cisco_8000()
```
Comment (xumia) [inline platform/broadcom/docker-saiserver-brcm/syncd_init_common.sh:91]: Why adding cisco-8000 in platform/broadcom?

Comment (richardyu-ms) [reply platform/broadcom/docker-saiserver-brcm/syncd_init_common.sh:91]: copied the syncd_init_common.sh from syncd repo.
We will use some other way to copy it from syncd repo

Comment (richardyu-ms) [reply platform/broadcom/docker-saiserver-brcm/syncd_init_common.sh]: ok, copy it automatically.

**Review by xumia — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Comment by liuh-80** [comment]
202012 branch PR merged: https://github.com/sonic-net/sonic-buildimage/pull/13226

**Pull request merged**